packages/core/handle/handle.js

```diff
@@ -666,9 +676,8 @@ const call = function (method) {
  * ```
  *
  * @method   adaptEvent
- * @param    {Function}  adapter  Function to adapt the event payload
+ * @param    {EventAdapter}  adapter  Function to adapt the event payload
  * @param    {Function}  handler  Handler to call with the new event payload
- * @param    {...*}      [args]   Additional args passed to both `adapter` and `handler`
  *
  * @returns  {HandlerFunction}    New event payload
```

Comment:
We should update the description to describe that it returns a handler function.  Possibly worth saying that it returns the return value of the handler when invoked.